src/transaction/transaction-box.test.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { create } from 'react-test-renderer';
+import { TransactionBox } from './transaction-box';
+
+test('should show Transaction Box with ETH Icon', () => {
```

Comment:
I've been using the most commonly accepted syntax `describe() -> it()` let's stick to it

Reply:
also, test file name should be '.spec.tsx'

Reply:
@sk91 yeap, I was wondering about that as well but when I went to the jest documentation they stop using `describe() -> it()` so I think for good reference for devs I think we should align our tests with the documentation. Thoughts?